src/app/portlets/content-types/main/content-types.component.ts

```diff
@@ -123,13 +128,21 @@ export class ContentTypesPortletComponent implements OnInit {
         this.listing.loadFirstPage();
     }
 
+    private setFilterByContentType(contentType: string) {
+        this.filterBy = contentType;
+        this.listing.paginatorService.setExtraParams('type', this.filterBy);
+        this.actionHeaderOptions.primary.model = this.actionHeaderOptions.primary.model.filter(
+            (item: ButtonModel) => item.label === this.filterBy
+        );
+    }
+
     private getPublishActions(pushPublish: boolean, addToBundle: boolean): DotDataTableAction[] {
         const actions: DotDataTableAction[] = [];
         /*
             Only show Push Publish action if DotCMS instance have the appropriate license and there are
             push publish environments created.
         */
-       if (pushPublish) {
+        if (pushPublish) {
```

Comment:
Remove space.

Reply:
for some reason it looks like a space added, but it isn't
https://github.com/dotCMS/core-web/blob/b3d021c1708197b787c402d5d97f090020061794/src/app/portlets/content-types/main/content-types.component.ts#L144